android/engine/src/test/resources/plans/child-immunization-schedule/structure-map-register.txt

```diff
@@ -43,6 +44,22 @@ group ExtractTask(
                subject -> groupIdentifier.use = "secondary" "rule_group_identifier_use";
 
         } "rule_group_identifier";
+
+         subject -> task.partOf = create('Reference') as ref then {
+                subject -> evaluate(questionnaireResponse, questionnaireResponse.descendants().where(linkId='123456').answer.valueString) as taskId then{
```

Comment:
@dubdabasoduba  I am not able to evaluate questionnaireResponse Can you please check this?